### PR TITLE
Add domain-offensive as dns-01 challenge

### PR DESCRIPTION
**Pull request by SvenLie**
This adds dns-01 challenge for domain-offensive (do.de)

### CODE REVIEW

**Comment by SvenLie** [comment]
I fixed the ansible lint stuff. :)

**Comment by avalor1** [comment]
Thank you @SvenLie, the integration tests are currently failing, did you test your code somewhere? The assert of the certificate fails.

**Comment by SvenLie** [comment]
> Thank you @SvenLie! The integration tests are currently failing, did you test your code somewhere? The assert of the certificate fails.

Hi @avalor1 yeah i tested my code and the certificate was valid. The integration test in the pipeline only test pebble or have i misunderstood something?

File: docs/dns-challenge/domain-offensive.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Variables for Domain Offensive dns-challenge
+
+| Variable                | Required | Default | Description
+|-------------------------|----------|---------|------------
+| acme_dns_password       | yes      |         | Let`s Encrypt API-Token, you can get here: [do.de](https://my.do.de/settings/domains/general)
```
Comment (rndmh3ro) [inline docs/dns-challenge/domain-offensive.md:5]:
```suggestion
| acme_dns_password       | yes      |         | Let's Encrypt API-Token, you can get here: [do.de](https://my.do.de/settings/domains/general)
```

**Comment by rndmh3ro** [comment]
Can you please rebase your PR?

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**